lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_corrupt_stateless_mme.py

```diff
@@ -49,56 +49,49 @@ def test_attach_detach_with_corrupt_stateless_mme(self):
             "UE id ", req.ue_id,
         )
 
-        for s in services_state_dict:
-            # Now actually complete the attach
```

Comment:
`services_state_dict` definition in lines 42-44 can be removed, as well.

Reply:
done